src/plugins/dna_export/src/ExportTasks.cpp

```diff
@@ -197,6 +203,9 @@ void ExportMSA2MSATask::run() {
             }
             resseq.replace("*", "X");
             DNASequence rs(name, resseq, aminoTT->getDstAlphabet());
+            if (reverseComplement) {
```

Comment:
This is wrong. Reversed-only sequences have no sense/meaning.
You already did reverse-complement above and it is enough.

Reply:
The meaning of this is to turn from left-to-right to right-to-left representation as it's done in sequence view (the reverse-complement strand in SV is written from right to left). I can remove this if you want.

Reply:
Amino sequences have only 1 reading direction. Even in the sequence view if you copy the rev-compl translation you will have a 'one-way' (left-to-right) amino sequence.

I believe the reverse here is not needed and should be removed.

Reply:
OK, fixed